docs/modules/getting-started/pages/wan.adoc

```diff
@@ -0,0 +1,282 @@
+. You need to provide your license key; this is needed since you are going to use the WAN Replication feature for the `tokyo` cluster in Management Center.
+Click on **View Cluster** for `tokyo`, go to **Settings** located on the very top right of the user interface and, select **License**.
++
+image:wan/provide-license.png[Enter your license key]
+Type in your license key and click on **Update License**. Close the license screen.
+. Verify that the `tokyo` cluster has WAN Replication enabled. Go to **Cluster > WAN Replication**.
++
+image:wan/verify-wan-replication.png[Verify that Tokyo cluster has WAN Replication feature enabled]
+
+== Step 4. Create a Map
+
+In this step you switch to the SQL shell in a terminal, create a map called `cities` for the `tokyo` cluster, and put data into it.
+
+. In a new terminal, start the SQL shell that will be connected to the `tokyo` cluster.
++
+[source,shell,subs="attributes+"]
+----
+docker run --network hazelcast-network -it --rm hazelcast/hazelcast:{full-version} hz-cli --targets tokyo@172.18.0.4:5701 sql
+----
+. Once you see the SQL shell (`sql>`), type the following command and press Enter to create the map.
++
+[source,shell]
+----
+CREATE MAPPING cities (__key INT, country VARCHAR, city VARCHAR)
+TYPE IMap 
+OPTIONS ('keyFormat'='int', 'valueFormat' = 'json-flat');
+----
+. Then, type the following command and press **Enter** to add data to the map.
++
+[source,shell]
+----
+INSERT INTO cities VALUES
+(1,'Australia','Canberra'),
+(2,'Croatia','Zagreb'),
+(3,'Czech Republic','Prague'),
+(4,'England','London'),
+(5,'Turkey','Ankara'),
+(6,'United States','Washington, DC');
+----
+. See the entries by running the following query.
++
+[source,shell]
+----
+SELECT * FROM cities;
+----
+
+NOTE: See xref:sql:get-started-sql.adoc[Get Started with SQL over Maps] for more querying options using SQL on maps.
+
+You can also see the map and its entries in Management Center using SQL browser:
+
+. Select *tokyo* in the dropdown field left to *Cluster Connections* on top of the user interface.
+. Go to **Storage** > **Maps**, you will see the `cities` map information.
++
+image:wan/map-tokyo.png[Cities map in Tokyo cluster]
+. Click on **SQL Browser** located on the very top right of the user interface and choose `cities` in the *select a map* field.
+The SQL browser then shows the default query in its editor, `SELECT * FROM cities;`. 
+. Click on **Execute Query**; you will see the data you've put in. 
++
+image:wan/map-entries.png[Management Center's SQL Browser shows the map data]
+. Close the SQL browser.
+
+== Step 5. Verify the Replication
+
+With WAN Replication enabled, your `cities` map and its data should have been replicated from the active cluster (Tokyo)
+to the passive one (London). In this step, you'll verify that the `cities` map now also exists in the London cluster.
+
+. In Management Center, select `london` in the dropdown field left to "Cluster Connections" on top of the user interface.
```

Comment:
```suggestion
. In Management Center, select *london* in the dropdown field left to "Cluster Connections" on top of the user interface.
```

UI text should always be bold.

Reply:
Please check the rest of the docs for instances where we refer to clicking UI elements. These should be in bold.